src/main/java/com/cedrus/aeolion/kafkaspringpong/kafka/SpringPongProducer.java

```diff
@@ -34,7 +34,7 @@ public void sendMessage(Message message) throws JsonProcessingException {
         kafkaProps.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, serializer);
         kafkaProps.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, serializer);
         Producer<String, String> producer = new KafkaProducer<>(kafkaProps);
```

Comment:
Maybe move the creation of the Producer to a private method.  Public methods should be a series or calls to other methods where possible.  Just makes it cleaner.